packages/graphql_flutter/lib/src/widgets/subscription.dart

```diff
@@ -68,11 +68,12 @@ class Subscription<TParsed> extends HookWidget {
 
   @override
   Widget build(BuildContext context) {
-    final client = useGraphQLClient();
+    final client = useGraphQLClient(); 
```

Comment:
```suggestion
    final client = useGraphQLClient();
```

looks like that the space is not needed?